Content.Shared/IgnitionSource/EntitySystems/SharedMatchstickSystem.cs

```diff
@@ -0,0 +1,102 @@
+using Content.Shared.Interaction;
+using Content.Shared.Item;
+using Content.Shared.Smoking;
+using Content.Shared.Temperature;
+using Robust.Shared.Audio.Systems;
+using Content.Shared.IgnitionSource.Components;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.IgnitionSource.EntitySystems;
+
+public abstract class SharedMatchstickSystem : EntitySystem
```

Comment:
```suggestion
public sealed partial class MatchstickSystem : EntitySystem
```
since they are empty we can just make this non-abstract and remove the server and client systems now

Reply:
I actually left them like that on purpose so it would be easier for forks to make changes if they wanted (E.g they wouldn't have to make a new class and it would reduce merge conflicts) - we don't really seem to have a policy either way and I'm also unsure if any forks even would have any changes. Let me know what you think!

Reply:
you can still do that since it is partial

Reply:
> you can still do that since it is partial

From what I remember (I could be wrong!) it doesn't quite work because the shared class isn't abstract so when you make a child it gets all funky. It doesn't matter much either way though so I'll just remove em! It's usually done that way anyway